src/Client/Core/TerminateInstanceOptions.cs

```diff
@@ -0,0 +1,31 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+namespace Microsoft.DurableTask.Client;
+
+/// <summary>
+/// Options to terminate an orchestration.
+/// </summary>
+public record TerminateInstanceOptions
```

Comment:
Can simplify this type as well:

``` CSharp
public record TerminateInstanceOptions(object? Output = null, bool Recursive = false)
```

Also, we need to keep recursive as default `false`. Otherwise, the default is a breaking behavior change from what we have today (which is non-recursive).

Reply:
Done